packages/@sanity/desk-tool/src/getIntentState.ts

```diff
@@ -40,8 +40,12 @@ export function getIntentState(
         index: i,
       }) ||
       // OR
-      // the pane is a documentList with a matching schemaTypeName
-      (pane.type === 'documentList' && pane.schemaTypeName === params.type)
+      // if the resolved pane is a document list
+      (pane.type === 'documentList' &&
+        // and the schema type matches
+        pane.schemaTypeName === params.type &&
+        // and the filter is the default filter
```

Comment:
note from espen review:

add comment for rationale: want to have decent default matches but don't want to match too heavily. aka false positives etc

if the user wants the match, they can implement canHandleIntent themselves